### PR TITLE
Add unmark task feature

### LINKED ISSUE #77: As a user, I can unmark a task
... so that I can set it to incomplete if there was an accidental update/mark

### PR DESCRIPTION
**Pull request by wxy1203**
### Changes ###
- UnmarkTaskCommand, UnmarkTaskCommandParser, TaskPanelParser
- User Guide, Developer Guide

Closes #77

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -126,6 +126,15 @@ Format: `task mark TASK_INDEX`
 Examples:
 * `task list` followed by `task mark 1` marks the 1st task in the task list as complete.
 
+### Unmarking tasks of a person: `unmark`
```
Comment (kianseong) [inline docs/UserGuide.md]: Should this be `task unmark`?

**Review by kianseong — APPROVED** [review]
LGTM!

**Review by amresh2000 — COMMENTED** [review]
Add the tests and it will be ready to merge

**Review by yilinzyl — APPROVED** [review]
LGTM!

File: docs/UserGuide.md (hunk 1 above)
Comment (wxy1203) [reply docs/UserGuide.md]: Changed.

**Pull request merged**